Makefile

```
@@ -230,7 +230,7 @@ shell: ## connect to database shell
 # -- Database
 
 dbshell: ## connect to database shell
-	docker-compose exec app-dev python manage.py dbshell
+	docker compose exec app-dev python manage.py dbshell
```

```suggestion
	@$(COMPOSE_EXEC_APP) python manage.py dbshell
```

We can't use this here ?